pkg/router/f5/f5.go

```diff
@@ -461,6 +481,68 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 // Routines for controlling F5.
 //
 
+// ensureVxLANTunnel sets up the VxLAN tunnel profile and tunnel+selfIP
+func (f5 *f5LTM) ensureVxLANTunnel() error {
+	glog.V(4).Infof("Checking and installing VxLAN setup")
+
+	// create the profile
+	url := fmt.Sprintf("https://%s/mgmt/tm/net/tunnels/vxlan", f5.host)
+	profilePayload := f5CreateVxLANProfilePayload{
+		Name:         "vxlan-ose",
```

Comment:
Can we use a constant for `vxlan-ose`? Might be a candidate for a parameter name, maybe?